jdbcSource/README.md

```diff
@@ -104,6 +107,12 @@ The Configuration document may look similar to the following example:
 *   **username**: Required. This is the username that will be used to connect to the SQL Database.
 *   **password**: Required. This is the password that will be used to connect to the SQL Database.
 *   **dbURL**: Required. This is the URL corresponding to the SQL Database that you will connect to.
+*   **asynchronousProcessing**: Optional. If set to `true`, query and publish requests will be handled asynchronously, as 
+opposed to the default behavior which is synchronous.
+*   **maxRunningThreads**: Optional. Only used if `asynchronousProcessing` is set to `true`. The maximum number of threads 
+running at any given point for query or publish requests, respectively. Must be a non-zero integer. Default value is 5.
+*   **maxQueuedTasks**: Optional. Only used if `asynchronousProcessing` is set to `true`. The maximum number of queued 
```

Comment:
Both of these must be a _positive_ integer. 

Separate:  are queue tasks running?  That is, is there a relationship between the two numbers?  Can `maxQueuedTasks` be zero, meaning if there are no runners, throw an error?

(Not a request to provide that, just want to know)